rewrite-maven/src/main/java/org/openrewrite/maven/RemoveRedundantDependencyVersions.java

```diff
@@ -346,4 +387,16 @@ public Xml.Tag visitTag(Xml.Tag tag, ExecutionContext ctx) {
             return t;
         }
     }
+
+    private static class RemoveEmptyPluginsTags extends MavenIsoVisitor<ExecutionContext> {
+        @Override
+        public Xml.Tag visitTag(Xml.Tag tag, ExecutionContext ctx) {
+            Xml.Tag t = super.visitTag(tag, ctx);
+            if (("pluginManagement".equals(t.getName()) || "plugins".equals(t.getName())) && (t.getContent() == null || t.getContent().isEmpty())) {
```

Comment:
It seems `t.getContent().isEmpty()` returns false if there is just a comment left, do we want to keep it then? (but I guess it’s the same for `<dependencyManagement>`)

Reply:
Comments are typically attached to the next element, such that when the `<plugin>` is removed, so is any comment attached to it. As such I don't think we have to handle the case of a dangling comment here, although I could be wrong. Thanks for pointing it out though! I think it's rare enough that we don't need to handle this just yet.